src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        int result;
+        result = ((result1 == bet1) && (result2 == bet2)) ? 2 :
```

Comment:
Good start, but try to think how you can make solution less complicated.
Hint:
you don't need separate variable "result"

Reply:
I don't know how to do it without variable and if-else construction(
Maybe you can help me to find more information about trenor operator?
Thnak you in advance